### ISSUE TITLE
`clickHandler` is a bad name

### ISSUE THREAD

**Issue by amcdnl** [swimlane/ng2d3#120]
`clickHandler` is the click event for when clicking stuff in the charts. Its a bad name per the advised spec, it should be something like `(select)="onSelect($event)"` and then later `(activate)="onActivate($event)"` for keyboard navigation.
